go.mod

```diff
@@ -1,8 +1,11 @@
 module github.com/rgl/packer-provisioner-windows-update
 
 require (
-	github.com/hashicorp/packer v1.4.0
+	github.com/hashicorp/packer v1.4.5
```

Comment:
not sure this is the correct way to fix the dependency issue I saw

```
go: github.com/hashicorp/packer@v1.4.0 requires
        github.com/gophercloud/gophercloud@v0.0.0-20180903124057-ea7289ebdf06: invalid pseudo-version: does not match version-control timestamp (2018-09-17T09:41:35Z)
```

Reply:
I've issued the command `go mod tidy` to tidy things up.